MiniProject1-master/Holmes_Mini_Project_1.py

```diff
@@ -0,0 +1,89 @@
+from pattern.en import sentiment
+from pattern.en import modality
+
+def sort_data(data):
```

Comment:
I recommend commenting functions - something along the lines of, "put in _______ and get out _______".